models/search-filter-params.ts

```diff
@@ -79,6 +79,7 @@ class SearchFilterParams {
       est_bio: this.params.label === 'bio',
       egapro_renseignee: this.params.label === 'egapro',
       est_ess: this.params.label === 'ess',
+      est_organisme_formation: this.params.label === 'estOrganismeFormation',
```

Comment:
peut-être préférer un slug plus court ? `of`ou même `qualiopi`

Par ailleurs il faut aussi mettre a jour le `structureLabels` L182